### PR TITLE
Fix a text input bug on TV

### PR DESCRIPTION
**Pull request by bbrto21**
* Check the device where the key event occurred

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Comment by bwikbs** [comment]
First of all, I'm sorry... This seems to be my original [sin.](https://github.com/bwikbs/engine/commit/d07723df91e66d761fd9278f903789d74fba5037#diff-5e915f93ae49530b09e296217ce99094a0fa238cf5cba8829a44384c61f6b0eeR383) 

However, the reason for using such logic was not to use ``ecore_device_name_get``... AKAIK,It's rejected my mobile store.

File: shell/platform/tizen/channels/text_input_channel.cc [hunk 1]
```diff
@@ -402,12 +402,11 @@ void TextInputChannel::SendStateUpdate(const flutter::TextInputModel& model) {
 bool TextInputChannel::FilterEvent(Ecore_Event_Key* keyDownEvent) {
   bool handled = false;
 
-#ifdef TIZEN_RENDERER_EVAS_GL
+#ifdef WEARABLE_PROFILE
   // Hardware keyboard not supported when running in Evas GL mode.
```
Comment (swift-kim) [inline shell/platform/tizen/channels/text_input_channel.cc:406]: nit:
```suggestion
  // Hardware keyboard not supported on watches.
```

**Review by bwikbs — APPROVED** [review]
Thanks!

File: shell/platform/tizen/channels/text_input_channel.cc [hunk 2]
```diff
@@ -399,72 +399,71 @@ void TextInputChannel::SendStateUpdate(const flutter::TextInputModel& model) {
...
-  ecoreKeyDownEvent.modifiers =
-      EcoreInputModifierToEcoreIMFModifier(keyDownEvent->modifiers);
-  ecoreKeyDownEvent.locks =
-      EcoreInputModifierToEcoreIMFLock(keyDownEvent->modifiers);
-  ecoreKeyDownEvent.dev_name = isIME ? "ime" : "";
-  ecoreKeyDownEvent.keycode = keyDownEvent->keycode;
-
-  if (isIME && strcmp(keyDownEvent->key, "Select") == 0) {
+  Ecore_IMF_Event_Key_Down ecore_key_down_event;
+  ecore_key_down_event.keyname = key_down_event->keyname;
```
Comment (swift-kim) [inline shell/platform/tizen/channels/text_input_channel.cc:413]: How about simply
```suggestion
  imf_event.keyname = event->keyname;
```

**Review by swift-kim — APPROVED** [review]
Thank you!

File: shell/platform/tizen/channels/text_input_channel.cc (hunk 2 above)
Comment (bbrto21) [reply shell/platform/tizen/channels/text_input_channel.cc]: Thanks for the nice comment!

**Pull request merged**